Audit Logging Support on Sync Gateway (#803)
couchbaselabs/docs-sync-gateway: modules/ROOT/pages/data-modeling.adoc

include::partial$_set_page_context.adoc[]

// BEGIN -- Page Attributes
:xref-sdk-nodejs-pg-start: xref:nodejs-sdk:hello-world:start-using-sdk.adoc[Node.js Server SDK]
:xref-sdk-java-encryption: xref:java-sdk:concept-docs:encryption.adoc#format[field encryption format]
// END -- Page Attributes

// BEGIN -- Page Heading
:param-abstract!:
:param-related:  {revisions--xref} | {managing-tombstones--xref}
include::partial$_show_page_header_block.adoc[]
// END -- Page Heading

== Introduction

> The guidance and constraints documented here relate to the design of data buckets and documents that you require, or may potentially require, to replicate using Sync Gateway functionality.

Reviewer: Simplify

```suggestion
This page includes guidance and constraints relating to the design of data buckets and documents that you want to replicate using Sync Gateway.
```

Author: Done. I'm not sure I touched these pages though?

When you might encounter the error::
You may encounter the error in the following deployment situations:

* In Mobile-to-Web Data Sync with {xref-sdk-nodejs-pg-start} and http://ottomanjs.com/[Ottoman.js] (the Node.js ODM for Couchbase), where the rule conflicts with the `+_type+` property that is automatically added by _Ottoman.js_.
+
A suggested workaround in this scenario is to fork the _Ottoman.js_ library, perform a search-replace for the `+_type+` property and replace it without a leading underscore.

[NOTE] 
====
For versions 2.x of Sync Gateway, you can encounter the following error:

> * In Mobile-to-Web Data Sync with Field-level Encryption enabled, because the rule conflicts with the default {xref-sdk-java-encryption}

Reviewer: ```suggestion
* In Mobile-to-Web Data Sync with Field-level Encryption enabled, because the rule conflicts with the default {xref-sdk-java-encryption}.
```

Author: Done
